pkg/server/server.go

```diff
@@ -1603,8 +1604,15 @@ func (s *Server) PreStart(ctx context.Context) error {
 		}
 	})
 
+	// After setting modeOperational, we can block until all stores are fully
+	// bootstrapped.
 	s.grpc.setMode(modeOperational)
 
+	// Block until all stores are fully bootstrapped.
+	log.Infof(ctx, "waiting for all stores to be fully bootstrapped")
```

Comment:
I'd actually remove this logging statement as well. It's a bit confusing if you consider that we'll also be logging this when restarting nodes with already bootstrapped stores (from the very first start). So, why are we waiting for anything?